guides/common/modules/proc_creating-a-remote-file-repository.adoc

```diff
@@ -25,20 +25,36 @@ endif::[]
 
 To create a file type repository in a remote directory, complete the following procedure:
 
-. On your remote server, ensure that the Server and {project-client-name} repositories are enabled.
 ifdef::satellite[]
+. On your machine, ensure that the right repositories are enabled.
+** For {EL} 8:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# subscription-manager repos --enable={RepoRHEL8BaseOS} \
+--enable={RepoRHEL8AppStream} \
+--enable={RepoRHEL8ServerSatelliteUtils}
+----
+** For {EL} 7:
 +
 [options="nowrap" subs="+quotes,attributes"]
 ----
 # subscription-manager repos --enable={RepoRHEL7Server} \
---enable={project-client-RHEL7-url}
+--enable={RepoRHEL7ServerSatelliteUtils}
 ----
 endif::[]
 . Install the Pulp Manifest package:
+** For {EL} 8:
 +
-[options="nowrap" subs="+quotes"]
+[options="nowrap" subs="+quotes,attributes"]
+----
+# {package-install-project} python3-pulp_manifest
+----
+** For {EL} 7:
++
+[options="nowrap" subs="+quotes,attributes"]
 ----
-# yum install python3-pulp_manifest
+# {package-install-project} tfm-pulpcore-python3-pulp_manifest
```

Comment:
I am not completely sure, but IMHO `package-install-project` depends on the project version, aka. its "dnf" for the most recent versions of Foreman, and "yum" for older versions. See https://github.com/theforeman/foreman-documentation/blob/master/guides/common/attributes-base.adoc#L67

Since you have to run the procedure on your local machine (not your Foreman Server), I guess using "yum" for EL7 and "dnf" for EL8 is better than relying on the attribute here. Other thoughts @Lennonka ?

Reply:
I agree `yum` is the correct suggestion here.
```suggestion
# yum tfm-pulpcore-python3-pulp_manifest
```